### ISSUE TITLE
Ability to set a primary MFA method

### ISSUE THREAD

**Issue by brynwhyman** [Firesphere/silverstripe-bootstrapmfa#15]
As a CMS user who has set up MFA for the first time, I want this method to be set as my primary MFA method, so that I am presented with this option as the default on login (if multiple methods are later stored).

AC
- [ ] The first MFA method to be set up becomes the user's primary method
- [ ] The primary method flag is built in a way so that the user can update their primary method

Notes:
- Updating a primary method within the CMS will be captured in another issue

**Comment by ScopeyNZ** [comment]
What's the purpose of registering other MFAs? Are we supposed to provide those as a fallback if the primary MFA validation cannot be done for some reason?

**Comment by robbieaverill** [comment]
FYI I have Google Auth and Yubikey set up on one of my MFA enabled sites. It usually asks me to enter my key first, with a small link saying “click here to use an Authenticator app” or something - I can switch between configured MFA options

**Comment by newleeland** [comment]
> What's the purpose of registering other MFAs? Are we supposed to provide those as a fallback if the primary MFA validation cannot be done for some reason?

Yes, experience wise, using another MFA method is usually much faster at logging the user in than, using recovery codes or calling their Admin, if they get locked out of their primary method. It also lowers the chance of platform users needing Ops.
